packages/mds-logger/index.ts

```diff
@@ -33,18 +33,24 @@ const getCustomProps = () => {
 }
 
 const log =
-  (level: LogLevel, namespace: string) =>
+  (level: LogLevel, namespace: string, redactKeys: Set<string>) =>
   (message: string, data?: Record<string, unknown> | Error): void => {
     if (process.env.QUIET !== 'true') {
       return data
-        ? logger[level]({ ...getCustomProps(), namespace, data: redact(data), message })
+        ? logger[level]({ ...getCustomProps(), namespace, data: redact(data, redactKeys), message })
         : logger[level]({ ...getCustomProps(), namespace, message })
     }
   }
 
-export const createLogger = (namespace: string, options: Partial<{ debugPrefix: string }> = {}) => ({
-  debug: debugLog(`${options.debugPrefix ?? 'mds'}:${namespace}`),
-  info: log('info', namespace),
-  warn: log('warn', namespace),
-  error: log('error', namespace)
-})
+export const createLogger = (
+  namespace: string,
+  { debugPrefix, redactKeys = [] }: Partial<{ debugPrefix: string; redactKeys: string[] }> = {}
+) => {
+  const keys = new Set(['lat', 'latitude', 'lng', 'lon', 'longitude', ...redactKeys.map(normalize)])
```

Comment:
Might be redundant now, but would be good in case someone added `'ID'` or `'access_token'` or something else to the standard list of keys without understanding that keys are all normalized for comparison.
```suggestion
  const keys = new Set(['lat', 'latitude', 'lng', 'lon', 'longitude', ...redactKeys].map(normalize))
```

Reply:
No unnecessary footguns!